src/MongoDB.Driver.Core/Core/Operations/ChangeStreamCursor.cs

```diff
@@ -88,7 +88,7 @@ public bool MoveNext(CancellationToken cancellationToken = default(CancellationT
             }
             catch (Exception ex)
             {
-                if (CanResumeAfter(ex))
+                if (RetryabilityHelper.IsResumableChangeStreamException(ex))
```

Comment:
The logic for which exceptions are resumable and retryable is centralized in the new RetryabilityHelper class.